cmd/containerd-release/main.go

```diff
@@ -94,7 +94,7 @@ This tool should be ran from the root of the project repository for a new releas
 		if err != nil {
 			return err
 		}
-		logrus.Info("Welcome to the %s release tool...", r.ProjectName)
+		logrus.Info("Welcome to the release tool...", r.ProjectName)
```

Comment:
any reason to not just change `Info` to `Infof`? having the project name prior to `release tool...` will be clearer IMO

Reply:
@estesp agree . Will resend the patch . thanks for the review.